crates/spicepod/Cargo.toml

```diff
@@ -19,3 +19,4 @@ duckdb = []
 postgres = []
 sqlite = []
 mysql = []
+clickhouse = []
```

Comment:
These features aren't used anymore in `crates/spicepod` - so we don't need to add this here.

Reply:
Fixed